src/main/webapp/app/course/manage/overview/course-management-overview-dto.model.ts

```diff
@@ -0,0 +1,8 @@
+import { Exercise } from 'app/entities/exercise.model';
+
+export class CourseManagementOverviewDto {
+    public courseId: number;
```

Comment:
are you sure those values always exist? is this 100% guaranteed?

Reply:
Yes, we always explicitly set this on the server side.

Reply:
We can change it to use a `?` though.